### PR TITLE
issue-10-rename-clickme

### PR DESCRIPTION
**Pull request by CarlaLlama**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
     ],
     entry_points='''
         [console_scripts]
-        clickme=vumi_http_proxy.clickme:cli
+        queen_of_ni=vumi_http_proxy.queen_of_ni:cli
```
Comment (hodgestar) [inline setup.py]: We could also name the script `queen-of-ni` (just the LHS, not the Python module)? Not sure whether you like `-` or `_` more in command names?

File: vumi_http_proxy/http_proxy.py [hunk 2]
```diff
@@ -120,6 +121,8 @@ def read_file(self, blacklistfile):
         blacklist = []
         if not blacklistfile:
             log.err("No blacklist config file provided.")
+            log.err('Ending process.')
+            sys.exit()
```
Comment (hodgestar) [inline vumi_http_proxy/http_proxy.py]: Exiting is probably a good idea, but perhaps not here. I think the solution is to move reading the config file out of `http_proxy` and into a helper function that we can call from `servicemaker` or `queen_of_ni`. Maybe remove this from this branch and we can make a separate issue for exiting if no blacklist is given?

**Comment by CarlaLlama** [comment]
I'll make a new branch for the exiting

**Comment by CarlaLlama** [comment]
@hodgestar can I land this in the meantime? :)

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**